modules/post/multi/manage/screensaver.rb

```diff
@@ -45,7 +54,8 @@ def cmd_vexec(cmd)
   def lock_session
     case session.platform
     when 'linux'
-      cmd_vexec('xdg-screensaver lock')
+      cmd_vexec('xdg-screensaver lock') unless !command_exists?('xdg-screensaver-lock')
+      cmd_vexec('loginctl lock-session') unless !command_exists?('loginctl')
```

Comment:
There's a few permutations of these commands that we would want to handle. E.g.: 
- both commands are found, but both exit with a non-zero status code; this should likely be reported to the end-user as a failure.
- The first command succeeds, in which case, do we want to run the second command?
- Neither of the commands exist